lib/module.js

```diff
@@ -617,7 +617,16 @@ Module._initPaths = function() {
     homeDir = process.env.HOME;
   }
 
-  var paths = [path.resolve(process.execPath, '..', '..', 'lib', 'node')];
+  // $PREFIX/lib/node, where $PREFIX is the root of the Node.js installation.
+  var prefixDir;
+  // process.execPath is $PREFIX/bin/node except on Windows where it is 
```

Comment:
Ah sorry. I know I ran the linter before (which is why this comment is no longer a single line as it was >80) but I must have forgotten afterwards. I won't be able to fix this until tomorrow (only have my tablet with me ATM).